packages/editor/src/lib/editor/Editor.ts

```diff
@@ -212,9 +212,9 @@ export class Editor extends EventEmitter<TLEventMap> {
 		const shapeUtils = {} as Record<string, ShapeUtil>
 		const allStylesById = new Map<string, StyleProp<unknown>>()
 
-		for (const { util: Util, props } of allShapes) {
+		for (const { type, util: Util, props } of allShapes) {
```

Comment:
We're using the `type` of the shape util, as passed into `defineShape`, to do all the work that the static type used to do, right? That's great!

Reply:
yup!